src/Server/modules/WorldPlaceManager.js

```diff
@@ -37,7 +36,15 @@ function AddWorldData(province, country, date, tod, number) {
 				}
 				var temp_date = helper.stringToDate(date);
 				allWorldPlace[i].numConfirmed[temp_date] = number;
-				allWorldPlace[i].currentNumConfirmed = number;
+				// Here: numConfirmed has new date (temp_date) added
```

Comment:
This is new logic i came up with for this part. Please tell me it works T.T I tested it and it seems like its works.
I tested with ("Tasmania", "Australia", "1/14/2020", 1, 1133)
Results:
- Jan 14 case is added to numConfirmed with 1133 as num of cases at the BOTTOM of the numConfirmed dictionary
- currentNumConfirmed was unchanged

I tested again with ("Tasmania", "Australia", "4/15/2020", 1, 1133);
Results:
- Apr 15 case is added to numConfirmed with 1133 as num of cases at the BOTTOM of the numConfirmed dictionary
- currentNumConfirmed was changed to 1133

Reply:
Looks great!!! XD XD